tests/protocol/material/test.js

```diff
@@ -0,0 +1,58 @@
+if (typeof PATH_TO_THE_REPO_PATH_UTILS_FILE === 'undefined') PATH_TO_THE_REPO_PATH_UTILS_FILE = "https://raw.githubusercontent.com/highfidelity/hifi_tests/master/tests/utils/branchUtils.js";
+Script.include(PATH_TO_THE_REPO_PATH_UTILS_FILE);
+var nitpick = createNitpick(Script.resolvePath("."));
+
+nitpick.perform("Material protocol sanity - TEST REQUIRES SERVER", Script.resolvePath("."), "secondary", function(testType) {
+    Script.include('../common.js');
+    setup();
+    
+    var object;
+    var entityProperties = setCommonEntityProperties();
+
+    entityProperties.type ="Material";
+    
+    entityProperties.boundingBox = {
+        brn: { x: 1.0663206577, y: 3.33795213699, z: 5.55088996887 },
+        tfl: { x:  1.235045075416, y: 3.490031242370, z:  5.69143104553222 },
+        center: { x: 1.1506829261779785, y: 3.413991689682, z:  5.6211605072 },
+        dimensions: { x: 0.1687244176864624, y: 0.15207910537719727, z:  0.14054107666015625 }
+    };
+
+    entityProperties.queryAACube = { x: 1.0616825819015503, y: 3.2616827487945557, z: 5.461682319641113, scale: 0.27663490176200867 };
```

Comment:
This raised my eyebrow: why would our automated tests need to ever know about or mess with the `queryAACube` property.  I see... it needs to be done because our protocol includes it.

Why do I care?  I've made a proposal that would allow us to purge the `queryAACube` stuff.

Reply:
Exactly ... the tests are "dumb"